### PR TITLE
Fix docker-compose with pm2 usage

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do ?

The docker-compose stack provided with this repository was outdated.  

This PR update docker-compose stack : 
  - remove the proxy container
  - fix env variables
  - add PM2 support

### How should this be manually tested?

  - Step 1 : Get the branch: `git fetch origin fix-docker-compose && git checkout fix-docker-compose`
  - Step 2 : Run the docker compose stack: `docker-compose -f docker/docker-compose.yml up`
  - Step 3 : Edit `lib/index.js`, Kuzzle should automatically reload
  ...

### Other changes

 - typo
 - remove section about threadable plugin in README

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ class CorePlugin {
    *  - Kuzzle loads plugins in <kuzzle install dir>/plugins/enabled/* and try to instanciate them, also configuration and manifest.json file are read.
    *     /!\ Misconfiguration will stop starting sequance of kuzzle
    *  - Kuzzle tries to initialiate worker plugins in spawned process though pm2
-   *  - "init" function is called for both woker and regular plugins.
+   *  - "init" function is called for both worker and regular plugins.
```
Comment (ballinette) [inline lib/index.js]: There are no `worker` plugins anymore => we need to remove the precedent line + replace this one by `"init" function is called`

File: lib/index.js [hunk 2]
```diff
@@ -133,7 +133,7 @@ class CorePlugin {
      */
     this.routes = [
       {verb: 'get', url: '/say-something/:property', controller: 'myNewController', action: 'myNewAction'},
-      {verb: 'post', url: '/say-something', controller: 'myNewController', action: 'myNewAction'}
+      {verb: 'get', url: '/say-something', controller: 'myNewController', action: 'myNewAction'}
```
Comment (ballinette) [inline lib/index.js]: why ?
`post` method was a good example to show that we can pass arguments through uri placeholder in a `get`query, or within the body of a `post` query.

File: README.md [hunk 3]
```diff
@@ -38,15 +38,12 @@ This command will start a Kuzzle stack with this plugin enabled. To make develop
 You can choose to work on the Kuzzle development branch by defining the following environment variables before launching `docker-compose`:
 
 ```bash
-$ export KUZZLE_DOCKER_TAG=:develop
-$ export PROXY_DOCKER_TAG=:develop
+$ export KUZZLE_DOCKER_TAG=develop
```
Comment (scottinet) [inline README.md]: the `develop` tag is for kuzzle developer
I think that kuzzle plugin developers need a production environment

File: lib/index.js (hunk 2 above)
Comment (Aschen) [reply lib/index.js]: I forget to change it back, my bad

**Pull request merged**